.editorconfig

```diff
@@ -0,0 +1,14 @@
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.java]
+indent_size = 4
```

Comment:
+1 I have a preference for spaces ; FYI, it seems that google in its guidelines goes this way (https://google.github.io/styleguide/javaguide.html#s2.3.1-whitespace-characters), but the default eclipse (which might follow oracle official guidelines) uses tabs.